tests/kafkatest/tests/client/client_compatibility_features_test.py

```diff
@@ -109,7 +109,7 @@ def invoke_compatibility_program(self, features):
           raise
 
     @cluster(num_nodes=7)
-    @matrix(broker_version=[str(DEV_BRANCH)], metadata_quorum=quorum.all_non_upgrade)
+    @parametrize(broker_version=str(DEV_BRANCH))
```

Comment:
How about setting "describe-acls-supported" to `false`? We can still test other features for raft.